packages/spectral/src/types/Inputs.ts

```diff
@@ -346,5 +346,5 @@ export type InputFieldCollection = "valuelist" | "keyvaluelist";
 /** Config variable result collection */
 export type ConfigVarResultCollection = Record<
   string,
-  string | Schedule | Connection | JSONForm | ObjectSelection | ObjectFieldMap
+  string | Schedule | Connection | unknown | ObjectSelection | ObjectFieldMap
```

Comment:
```suggestion
  string | Schedule | Connection | JSONForm | ObjectSelection | ObjectFieldMap
```
Curious on the change from unknown to `JSONForm`

Reply:
The runtime value of a JSON Form config var is the values of the form instead of the form definition. We can't know what the form looks like so we type as `unknown`